Build Pipeline is broken on tags

Tag building is failing due to a missing dependency 

```


2 errors occurred:

--> linux/amd64 error: exit status 1

Stderr: main.go:7:2: cannot find package "github.com/vapor-ware/synse-sdk/sdk" in any of:

	/usr/local/go/src/github.com/vapor-ware/synse-sdk/sdk (from $GOROOT)

	/go/src/github.com/vapor-ware/synse-sdk/sdk (from $GOPATH)



--> darwin/amd64 error: exit status 1

Stderr: main.go:7:2: cannot find package "github.com/vapor-ware/synse-sdk/sdk" in any of:

	/usr/local/go/src/github.com/vapor-ware/synse-sdk/sdk (from $GOROOT)

	/go/src/github.com/vapor-ware/synse-sdk/sdk (from $GOPATH)


```

this looks like some dirty state or agent swapping happened that nuked the dep vendoring.

--- a/.jenkins
+++ b/.jenkins
@@ -69,7 +69,9 @@ pipeline {
       steps {
         script {
           docker.image('vaporio/jenkins-agent-golang:latest').inside('-v /var/run/docker.sock:/var/run/docker.sock -v /usr/bin/docker:/usr/bin/docker -e GITHUB_TOKEN=$GITHUB_TOKEN -v $WORKSPACE:$PROJ_PATH --group-add docker') {
-            sh 'cd ${PROJ_PATH} && goreleaser release --rm-dist'
+            withDockerRegistry(registry: [credentialsId: 'vio-docker-hub']) {
+              sh 'cd ${PROJ_PATH} && goreleaser release --rm-dist'
+            }
           }
         }
       }

